service/dap/server_test.go

```diff
@@ -74,13 +82,12 @@ func runTest(t *testing.T, name string, test func(c *daptest.Client, f protest.F
 	}()
 
 	client := daptest.NewClient(listener.Addr().String())
-	defer client.Close()
 
-	defer func() {
+	teardown := func() {
```

Comment:
I am actually removing code this in another PR: https://github.com/go-delve/delve/pull/2414/files#diff-7288f0435a51a46406796e0e110b3f8f1ea66e8c3e6b4209270c8f7f2ff89b9a
I don't think it is necessary (see comment). And even though there a stopOnce, I think it might have gotten flagged at one point for me with "go test -race".

Reply:
This PR got merged.

Reply:
Great! Merged it into this branch.